spec/test_files/get_commit_sha_4ceaceb20e700b92197daf8904f5c4e226625d8a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "total_count": 0,
+  "incomplete_results": false,
+  "items": [
+
+  ]
```

Comment:
You probably already verified this but just to confirm, this will be the expected result from Github's API if there is no PR associated with the corresponding commit right?

Reply:
yes, that's where I got the response from